pkg/webui/console/views/device-overview/index.js

```diff
@@ -223,6 +239,25 @@ class DeviceOverview extends React.Component {
 
     sheetData.push(sessionInfoData)
 
+    const macStateAndSettings = {
+      header: 'Export end device MAC state and MAC settings',
+      items: [],
+    }
+
+    if (Boolean(mac_state) && Boolean(mac_settings)) {
+      macStateAndSettings.items.push({
+        value: (
+          <Button
```

Comment:
```suggestion
          <ModalButton
```

Please use the modal button that will show a modal with a message reading either

1.  When the device has a session:

> The MAC data can contain sensitive information such as session keys that can be used to decrypt messages. **Do not share this information publicly.**

2\. When the device has no active session (so MAC state won't be available):

> The end device is currently not connected to the network (no active session). The MAC data will hence only contain the current MAC settings.